frontend/packages/console-dynamic-plugin-sdk/src/extensions/console-types.ts

```diff
@@ -145,6 +146,32 @@ export type PrometheusResponse = {
   warnings?: string[];
 };
 
+export type PrometheusPollProps = {
+  /** Delay between polling requests */
+  delay?: number;
+  /** One of the well-defined Prometheus API endpoints */
+  endpoint: PrometheusEndpoint;
+  namespace?: string;
+  /** Prometheus query */
+  query: string;
+  /** A search parameter */
+  timeout?: string;
+  /** A vector-query search parameter */
+  endTime?: number;
+  /** A vector-query search parameter */
+  samples?: number;
+  /** A vector-query search parameter */
+  timespan?: number;
+};
+
+export type UsePrometheusPoll = (
+  props: PrometheusPollProps,
+) => [
+  PrometheusResponse,
+  any /* Caught error. Can be originated by a request or internal processing. */,
+  boolean /* is loading? */,
```

Comment:
We should change the order to
```suggestion
  boolean,
  unknown
```

to align with [k8s watch hook](https://github.com/openshift/dynamic-plugin-sdk/blob/main/packages/lib-utils/src/k8s/hooks/watch-resource-types.ts#L9-L13). Also the comments do not bing any value IMO. Its better to add jsdoc to `usePrometheusPoll` itself

Reply:
done